designsystem/field/field.module.css

```diff
@@ -138,13 +148,17 @@
 		mask-size: 100%;
 	}
 	.field u-datalist {
+		opacity: 0;
+		transition: 125ms opacity ease-in-out;
```

Comment:
Don't know if we actually want this, I think it looks kinda nice, but I'm no designer.

The motivation wasn't really the transition, it was to wait showing the popover until its placed correctly. The first time it opens it was placed at `top: 0, left: 0`, flashing into view for a split second before moving to the correct place. This avoids this and adds a smooth fade.